### PR TITLE
Kraken: fix forbidden uri

### PR DESCRIPTION
**Pull request by xlqian**
http://jira.canaltp.fr/browse/ITI-29

### CODE REVIEW

**Comment by xlqian** [comment]
Jenkins, test this please!

File: source/routing/raptor.cpp [hunk 1]
```diff
@@ -628,6 +628,7 @@ void RAPTOR::set_valid_jp_and_jpp(
     // feasible ones.
     jpps_from_sp = data.dataRaptor->jpps_from_sp;
     jpps_from_sp.filter_jpps(valid_journey_pattern_points);
+
```
Comment (TeXitoi) [inline source/routing/raptor.cpp]: useless added line

File: source/routing/tests/raptor_test.cpp [hunk 2]
```diff
@@ -2932,3 +2932,78 @@ BOOST_AUTO_TEST_CASE(optimize_extention_before_min_wait) {
...
+    BOOST_REQUIRE_EQUAL(res2.size(), 1);
+    BOOST_REQUIRE_EQUAL(res2[0].items.size(), 4);
+
+    BOOST_CHECK_EQUAL(res2.at(0).items.front().departure, time_from_string("2016-Jul-22 08:11:00"));
+    BOOST_CHECK_EQUAL(res2.at(0).items.back().arrival, time_from_string("2016-Jul-22 09:27:00"));
+
+    BOOST_CHECK_EQUAL(res2.at(0).items.at(1).stop_points.front()->uri, chateletLesHalles);
+    BOOST_CHECK_EQUAL(res2.at(0).items.at(1).stop_points.back()->uri, chatelet);
+    BOOST_CHECK_EQUAL(res2.at(0).items.at(3).stop_points.front()->uri, chatelet);
+
```
Comment (TeXitoi) [inline source/routing/tests/raptor_test.cpp]: useless empty line

**Comment by TeXitoi** [comment]
OK for me.

**Pull request merged**